### PR TITLE
Mad u64 u32 only pre gfx1030

### PR DESCRIPTION
**Pull request by neon60**
-Disable injecting inline assembly
-Plus small CI changes.
-Readme update with link.

### CODE REVIEW

**Comment by saadrahim** [comment]
@neon60 This is not working on our CI. Also, please address @arsenm 's concerns.

**Comment by saadrahim** [comment]
note: !srcloc = 4145716

<inline asm>:1:26: error: invalid operand for instruction

 <inline asm> : 1 : 26 :  error:  invalid operand for instructionv_mad_u64_u32 v[18:19], s[6:7], s11, v13, v[4:5]


                v_mad_u64_u32 v[24:25], s[28:29], s1, v24, v[22:23] 

                                              ^ 

 note:  !srcloc =  4145716 

    ^

note: !srcloc = 4145716

<inline asm><inline asm>::11::2426: : error: error: invalid operand for instructioninvalid operand for instruction


                v_mad_u64_u32 v[3:4], s[6:7], s11, v19, v[4:5]v_mad_u64_u32 v[52:53], s[28:29], s5, v52, v[22:23]
 Is still showing up on our CI.

File: library/include/rocrand_common.h [hunk 1]
```diff
@@ -79,6 +65,8 @@ unsigned long long mad_u64_u32(const unsigned int x, const unsigned int y, const
     #endif
 }
 
+#undef OLDER_THAN_NAVI21
```
Comment (saadrahim) [inline library/include/rocrand_common.h:68]: This comment is not accurate and if we are dropping assembly completely, do we need it?

Comment (doctorcolinsmith) [reply library/include/rocrand_common.h:68]: I think the #undef was missed on the last PR update.

Comment (neon60) [reply library/include/rocrand_common.h:68]: I have to remove this.

Comment (neon60) [reply library/include/rocrand_common.h]: Removed this line.

**Comment by saadrahim** [comment]
Solves #140

**Pull request merged**